Fix load_model for multiple output metrics in dictionary

dieuwkehupkes opened the pull request:
Fixes the problem described in #3958 by adding an extra check in deserialized that recreates original dictionary.

isaacgerg commented:
Is this realted to #4143

fchollet commented on keras/models.py:
```diff
@@ -108,7 +108,15 @@ def deserialize(obj):
+                if type(value) is list:
+                    for element in value:
+                        if element in custom_objects:
+                            deserialized[key].append(custom_objects[value])
```
Should be `element` here...

fchollet commented:
Closing outdated PR. If you still care about the content of the PR, please submit a new PR to `master`, updated for the Keras 2.0 API.

dieuwkehupkes commented:
Dear Francois,

I submitted a new PR, the solution is still identical to the previous, but
updated for the Keras 2.0 API.

Best,

Dieuwke

2017-03-15 22:39 GMT+01:00 François Chollet <notifications@github.com>:

> Closing outdated PR. If you still care about the content of the PR, please
> submit a new PR to master, updated for the Keras 2.0 API.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/fchollet/keras/pull/4136#issuecomment-286887937>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/ACut4Cpdb0GIy9SoQPTM55_5VSUG_no9ks5rmFqGgaJpZM4KdE6F>
> .
>
